features/accounts/edit/profile-form/profile-form.tsx

```diff
@@ -64,77 +62,63 @@ export const ProfileForm = () => {
   return (
     <div className={style.formContainer}>
       <form className={style.form} onSubmit={handleSubmit(onSubmit)}>
+        {process.env.NEXT_PUBLIC_MODE === 'development' && <DevTool control={control} />}
```

Comment:
An example of using environment variable to conditionally render `DevTools` component.